Task1/Program.cs

```diff
@@ -10,7 +10,262 @@ class Program
+                    break;
+            }
+        }
+
+        public static void RangeOfNumber()
+        {
+            Console.WriteLine(" Enter number from 0 to 100");
+            int number = Convert.ToInt32(Console.ReadLine());
+            if (0 <= number && number <= 14)
+            {
+                Console.WriteLine("Number is in range from 0 to 14");
+            }
+            if (15 <= number && number <= 35 )
+            {
+                Console.WriteLine("Number is in range from 15 to 35");
+            }
+            if (36 <= number && number <= 50)
+            {
+                Console.WriteLine("Number is in range from 35 to 50");
+            }
+            if (50<= number && number <= 100)
+            {
+                Console.WriteLine("Number is in range from 51 to 100");
+            } 
+            if (0 > number || number > 100)
+                Console.WriteLine("Number is not in range from 0 to 100");
+        }
+
+
+        public static void Translator()
+        {
+            Console.WriteLine("Enter the word (мороз, солнце, дождь, тучи, ветер, гроза, туман, снег, жара, облака ):");
+            string word = Console.ReadLine();
+            switch (word)
+            {
+                case "мороз":
+                    Console.WriteLine("frost");
+                    break;
+                case "солнце":
+                    Console.WriteLine("sun");
+                    break;
+                case "дождь":
+                    Console.WriteLine("rain");
+                    break;
+                case "тучи":
+                    Console.WriteLine("clouds");
+                    break;
+                case "ветер":
+                    Console.WriteLine("wind");
+                    break;
+                case "гроза":
+                    Console.WriteLine("thunderstorm");
+                    break;
+                case "туман":
+                    Console.WriteLine("fog");
+                    break;
+                case "снег":
+                    Console.WriteLine("snow");
+                    break;
+                case "жара":
+                    Console.WriteLine("heat");
+                    break;
+                case "облака":
+                    Console.WriteLine("clouds");
+                    break;
+                default:
+                    Console.WriteLine("Undefined word");
+                    break;
+            }
+        }
+
+        public static void Premium()
+        {
+            Console.WriteLine(" Enter your salery:");
+            int salery = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine(" Enter your work experience");
+            int workExperience = Convert.ToInt32(Console.ReadLine());
+
+            if (workExperience <= 5)
+            {
+                Console.WriteLine($" Work experience is {workExperience}, salery is {salery}, premie is {(double)salery*0.1}");
```

Comment:
You could notice that this code is repeated several times.
Lets calculate bonus value is if-else statements and move CW to the end of method, to avoid repeating.